client/src/components/CreateContestForm.js

```diff
@@ -104,12 +104,34 @@ export default function CreateContestForm() {
             }
         })
     }
-
-    // temporary hardcode image names
-    const imageNames = ['tattoo-1.png', 'tattoo-2.png', 'tattoo-3.png', 'tattoo-4.png', 'tattoo-5.png', 
-    'tattoo-6.png', 'tattoo-7.png', 'tattoo-8.png', 'tattoo-9.png', 'tattoo-10.png', 
-    'tattoo-11.png', 'tattoo-12.png', 'tattoo-13.png', 'tattoo-14.png']
-
+    useEffect(() => { // Only runs once when first rendering
+        getInspirationalImages((data) => {
+            createGridListTiles(data) // Sets images equal to return from get request
+        }, (error) => {
+            // onError
+            if (error instanceof RequestError && error.response.status === 400) {
+                console.log(error.response.json())
+            } else {
+                console.log(error)
+            }
+        })
+    }, [])
+    const createGridListTiles = (data) => {
+        const imagesMap = new Map(Object.entries(data))
+        console.log(imagesMap)
+        const images = []
+        for (var i = 0; i < imagesMap.size; i++) {
+            const imageMap = new Map(Object.entries(imagesMap.get(String(i))))
+            const imageURL = imageMap.get('image_link')
+            const imageKey = imageMap.get('id')
+            images.push(
+                <GridListTile key={imageKey} cols={1}>
```

Comment:
you can probably simplify this code by just mapping through `data`

```
const = images [
  data.map(image => (
    <GridListTile key={image.id} cols={1}>
        <img src={image.image_link} alt={image.image_link} />
    </GridListTile>
  ))
]
```

Reply:
I just get "data.map" is not a function

Reply:
oh that's weird, is data not an array?

Reference on how React usually uses the map function https://github.com/hatchways/team-ninja-turtles/blob/dev/client/src/pages/DiscoveryPage.js#L100

regardless, this isn't too big of an issue. it would just make the code more clean